GVRf/Framework/src/org/gearvrf/asynchronous/AsyncBitmapTexture.java

```diff
@@ -217,9 +217,13 @@ private static Point getScreenSize(Context context, Point p) {
     private static void onGlInitialization() {
         if (glUninitialized) {
             int[] size = new int[] { -1 };
+            int errorCode = glGetError();
+            while(errorCode != GL_NO_ERROR) {
```

Comment:
* This loop isn't auto-formatted
* More importantly, what is the point of this loop? According to the OpenGL ES 3.0 Programming Guide, `glGetError()` "Returns the current error code and resets the current error coder to `GL_NO_ERROR`".